slothy/targets/aarch64/aarch64_neon.py

```diff
@@ -3175,7 +3176,50 @@ def core(inst,t,log=None):
 
     return core
 
-veor.global_fusion_cb  = eor3_fusion_cb()
+# TODO: Test only...
+# veor.global_fusion_cb  = eor3_fusion_cb()
+
+def eor3_splitting_cb():
```

Comment:
Please add a comment here what this is doing. 
Probably also worth pointing out that we dont actually use this in any crypto examples so far.